src/main/java/core/basesyntax/service/RegisterException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class RegisterException extends RuntimeException {
```

Comment:
It's better to rename to RegistrationException that describe that exception could be thrown during checking the parameters